src/core/TrustManagerExternalAttesterList.sol

```diff
@@ -22,8 +22,11 @@ abstract contract TrustManagerExternalAttesterList is IRegistry, TrustManager {
      */
     function check(address module, address[] calldata attesters, uint256 threshold) external view {
```

Comment:
@zeroknots do you think we should require sorted here? this might break some stuff so might be best not to? otherwise we'd need to sort them here imo

Reply:
yea I think it was open for discussion. but I think this would be a good fix